pkg/reconciler/ingress/resources/gateway.go

```diff
@@ -227,15 +242,11 @@ func GatewayRef(gw *v1alpha3.Gateway) tracker.Reference {
 	}
 }
 
-func makeIngressTLSGateway(ing *v1alpha1.Ingress, originSecrets map[string]*corev1.Secret, selector map[string]string, gatewayService *corev1.Service) (*v1alpha3.Gateway, error) {
+func makeIngressGateway(ing *v1alpha1.Ingress, selector map[string]string, servers []*istiov1alpha3.Server, gatewayService *corev1.Service) *v1alpha3.Gateway {
 	ns := ing.GetNamespace()
 	if len(ns) == 0 {
```

Comment:
sorry I did not get your point. Which part should not happen? I am not sure I understand `I guess the code still remains since ClusterIngress was used.` Could you please elaborate more?

Reply:
I think I understand it now. Basically we don't need to handle the case when namespace is empty as it is for ClusterIngress which has been deprecated. I updated the code to remove the logic.